eclair-core/src/test/scala/fr/acinq/eclair/api/JsonSerializersSpec.scala

```diff
@@ -37,6 +49,19 @@ class JsonSerializersSpec extends FunSuite {
 
     // but it works with our custom key serializer
     val json = Serialization.write(map)(org.json4s.DefaultFormats + new BinaryDataSerializer + new OutPointKeySerializer)
-    assert(json === """{"11418a2d282a40461966e4f578e1fdf633ad15c1b7fb3e771d14361127233be1:0":"dead","3d62bd4f71dc63798418e59efbc7532380c900b5e79db3a5521374b161dd0e33:1":"beef"}""")
+    assert(json === s"""{"${toBigEndian(tx1)}:0":"dead","${toBigEndian(tx2)}:1":"beef"}""")
```

Comment:
Good catch! embarassing...

Here is how I would fix it though (`OutPoint` already has the `hash`/`txid` conversion):

```scala
    val output1 = OutPoint("11418a2d282a40461966e4f578e1fdf633ad15c1b7fb3e771d14361127233be1", 0)
    val output2 = OutPoint("3d62bd4f71dc63798418e59efbc7532380c900b5e79db3a5521374b161dd0e33", 1)

    val map = Map(
      output1 -> BinaryData("dead"),
      output2 -> BinaryData("beef")
    )

    ...

    assert(json === s"""{"${output1.txid}:0":"dead","${output2.txid}:1":"beef"}""")
```